imagetool/src/main/resources/docker-files/Rebase_Image.mustache

```diff
@@ -0,0 +1,169 @@
+    JAVA_HOME={{{java_home}}} \
+    ORACLE_HOME={{{oracle_home}}} \
+    INV_LOC=${INV_LOC:-/u01/oracle/oraInventory} \
+    ORAINST=${ORAINST:-oraInst.loc} \
+    OPATCH_NO_FUSER=true
+
+    RUN mkdir -p {{{oracle_home}}} \
+    && chown {{userid}}:{{groupid}} {{{oracle_home}}}
+
+    # Install base WLS
+    {{#installJava}}
+        COPY --from=JDK_BUILD --chown={{userid}}:{{groupid}} {{{java_home}}} {{{java_home}}}/
+    {{/installJava}}
+
+    COPY --chown={{userid}}:{{groupid}} $WLS_PKG wls.rsp {{{tempDir}}}/
+    COPY --chown={{userid}}:{{groupid}} $ORAINST $INV_LOC/
+
+    {{#isOpatchPatchingEnabled}}
+        COPY --chown={{userid}}:{{groupid}} {{{opatchFileName}}} {{{tempDir}}}/opatch/
+    {{/isOpatchPatchingEnabled}}
+
+    {{#isPatchingEnabled}}
+        COPY --chown={{userid}}:{{groupid}} patches/* {{{tempDir}}}/patches/
+    {{/isPatchingEnabled}}
+
+    USER {{userid}}
+
+    {{#beforeFmwInstall}}
+        {{{.}}}
+    {{/beforeFmwInstall}}
+
+    RUN unzip -q {{{tempDir}}}/$WLS_PKG -d {{{tempDir}}} \
+    && {{{java_home}}}/bin/java -Xmx1024m -jar {{{tempDir}}}/fmw_*.jar -silent ORACLE_HOME={{{oracle_home}}} \
+    -responseFile {{{tempDir}}}/wls.rsp -invPtrLoc $INV_LOC/$ORAINST -ignoreSysPrereqs -force -novalidation \
+    {{#isOpatchPatchingEnabled}}
+        && cd {{{tempDir}}}/opatch \
+        && {{{java_home}}}/bin/jar -xf {{{tempDir}}}/opatch/{{{opatchFileName}}} \
+        && {{{java_home}}}/bin/java -jar {{{tempDir}}}/opatch/6880880/opatch_generic.jar -silent -ignoreSysPrereqs -force -novalidation oracle_home={{{oracle_home}}} \
+    {{/isOpatchPatchingEnabled}}
+    {{#isPatchingEnabled}}
+        && {{{oracle_home}}}/OPatch/opatch napply -silent -oh {{{oracle_home}}} -phBaseDir {{{tempDir}}}/patches \
+        && {{{oracle_home}}}/OPatch/opatch util cleanup -silent -oh {{{oracle_home}}} \
+    {{/isPatchingEnabled}}
+    && rm -rf {{{tempDir}}}
+
+    {{#afterFmwInstall}}
+        {{{.}}}
+    {{/afterFmwInstall}}
+
+
+    FROM OS_UPDATE as FINAL_BUILD
+
+    ARG ADMIN_NAME
+    ARG ADMIN_HOST
+    ARG ADMIN_PORT
+    ARG MANAGED_SERVER_PORT
+
+    ENV ORACLE_HOME={{{oracle_home}}} \
+    {{#installJava}}
+        JAVA_HOME={{{java_home}}} \
+    {{/installJava}}
+    LC_ALL=${DEFAULT_LOCALE:-en_US.UTF-8} \
+    PROPERTIES_FILE_DIR={{{oracle_home}}}/properties \
+    PATH=${PATH}:{{{java_home}}}/bin:{{{oracle_home}}}/oracle_common/common/bin:{{{oracle_home}}}/wlserver/common/bin:{{{oracle_home}}}
+
+    LABEL com.oracle.weblogic.imagetool.buildid="{{buildId}}"
+
+    {{#installJava}}
+        COPY --from=JDK_BUILD --chown={{userid}}:{{groupid}} {{{java_home}}} {{{java_home}}}/
+    {{/installJava}}
+
+    COPY --from=WLS_BUILD --chown={{userid}}:{{groupid}} {{{oracle_home}}} {{{oracle_home}}}/
+
+    USER {{userid}}
+    WORKDIR {{{work_dir}}}
+    RUN mkdir -p {{domain_home}}
+    COPY --from=SOURCE_IMAGE --chown={{userid}}:{{groupid}} {{domain_home}} {{domain_home}}/
+
+{{/isRebaseToNew}}
+
+
```

Comment:
The Domain's ports need to be exposed.  Since this is a new image, those "expose" commands are lost from the original.  Also, the rebase should set the WORKDIR